### ISSUE TITLE
Argotlunar VST (2.06) not recognized in Windows 10

### ISSUE THREAD

**Issue by gsenna** [mourednik/argotlunar#8]
Hi,

I tried Argotlunar on Linux(x64) with Reaper and it sounds great! However, I couldn't make Reaper recognize the VST on Windows 10(x64). Could there be a runtime library that maybe I'm missing or some other dependencies?

Thanks.

**Comment by mourednik** [comment]
Tested now with Windows 10 x64, Reaper 6.15. It works here. Maybe there is some old Visual C++ library missing on your system. 

Try installing both of these:
http://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU3/vcredist_x64.exe
http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x64.exe

**Comment by gsenna** [comment]
You were in the right track! It started being recognized after installing the 2010 sp1 redistributable. What's bothering me now is that I could swear I saw Reaper recognizing it once and then disappearing again after a simple Clear Cache/Rescan BEFORE installing any redistributable. But how could it have loaded without the C++ libraries? Anyway, I rescaned a couple of time since installing the aforementioned package and it seems the issue is now gone. 

Thanks again for the wonderful VST! Cheers.
